project/Docs.scala

```diff
@@ -47,8 +37,7 @@ object Docs {
       "gray-lighter" -> "#F4F3F4",
       "white-color" -> "#FFFFFF"),
     autoAPIMappings := true,
-    unidocProjectFilter in(ScalaUnidoc, unidoc) :=
-      inProjects(docsSourcesAndProjects(scalaVersion.value)._2: _*),
+    unidocProjectFilter in(ScalaUnidoc, unidoc) := inProjects(Seq.empty[ProjectReference]: _*),
```

Comment:
@kailuowang @LukaJCB I'm not sure this is necessary anymore, just wasn't sure. Thoughts?

Reply:
This setting should be removed. Otherwise if it will disable the scaladoc for the project